doc/conf.py

```diff
@@ -409,8 +327,8 @@ def findsource(object):
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-  ('index', 'xarray', u'xarray Documentation',
-   u'xarray Developers', 'xarray', 'One line description of project.',
+  ('index', 'xarray', 'xarray Documentation',
+   'xarray Developers', 'xarray', 'N-D labeled arrays and datasets in Python.',
```

Comment:
Yes, it does not appear as download option on RTD either. It's not very useful anyway...